Dockerfile-client

```diff
@@ -14,8 +14,12 @@ RUN yarn add dotenv
 RUN yarn run bootstrap
 RUN yarn run build
 
-RUN echo '*/1 * * * * cd /airnode && yarn run dev:invoke' > local-cron
+RUN echo '*/1 * * * * echo "AirNode Running"' > local-cron
 RUN /usr/bin/crontab /airnode/local-cron 
 
+HEALTHCHECK --interval=1m --timeout=50s --start-period=10s \
```

Comment:
This is pretty weird. The cron job is doing nothing, all the work is being done here, and it gives 10 seconds to the node to finish the job even though we assume the node can take 1 minute serving requests.